v2/cn/blog-understand-cosocket.md

```diff
@@ -1,11 +1,11 @@
 <!---
     @title         怎样理解 cosocket
-    @creator       覃冠日，李张勇
-    @created       2016-08-01 17:24 GMT
-    @modifier      覃冠日，李张勇
-    @modifier_link 
-    @modified      2016-07-30 17:24 GMT
-    @changes       1
+    @creator
+    @created
+    @modifier
+    @modifier_link
+    @modified
+    @changes
```

Comment:
You can keep the creator and created field names and values. But please remove the modifier, modier_link, and modified, and changes field *names* and *values* altogether. Please see `en/linux-packages.md` for such an example.